packages/server/package.json

```diff
@@ -0,0 +1,109 @@
+{
+	"$schema": "https://json.schemastore.org/package",
+	"name": "@svelte-docgen/server",
+	"version": "0.0.0",
+	"type": "module",
+	"description": "Allows creating virtual modules with docgen data for targetted Svelte component(s) - on demand.",
+	"keywords": [
+		"svelte",
+		"documentation",
+		"component",
+		"props",
+		"docgen",
+		"server"
+	],
+	"license": "MIT",
+	"contributors": [
+		{
+			"name": "Mateusz Kadlubowski",
+			"email": "xeho91@pm.me",
+			"url": "https://github.com/xeho91"
+		},
+		{
+			"name": "Taku Fukada",
+			"email": "naninunenor@gmail.com",
+			"url": "https://github.com/ciscorn"
+		}
+	],
+	"funding": [
+		{
+			"type": "github",
+			"url": "https://github.com/sponsors/svelte-docgen"
+		},
+		{
+			"type": "opencollective",
+			"url": "https://opencollective.com/svelte-docgen"
+		}
+	],
+	"homepage": "https://github.com/svelte-docgen/svelte-docgen",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/svelte-docgen/svelte-docgen.git",
+		"directory": "packages/vite-plugin-svelte-docgen"
+	},
+	"bugs": "https://github.com/svelte-docgen/svelte-docgen/issues",
+	"publishConfig": {
+		"access": "public",
+		"provenance": true
+	},
+	"engines": {
+		"node": ">=18"
+	},
+	"files": [
+		"src/",
+		"!src/**/*.test.*",
+		"types/"
+	],
+	"exports": {
+		"./bun": {
+			"types": "./types/mod.d.ts",
+			"default": "./src/bun/mod.js"
+		},
+		"./deno": {
+			"types": "./types/mod.d.ts",
+			"default": "./src/deno/mod.js"
+		},
+		"./node": {
+			"types": "./types/mod.d.ts",
+			"default": "./src/node/mod.js"
+		},
+		"./package.json": "./package.json"
+	},
+	"types": "./types/mod.d.ts",
+	"scripts": {
+		"build": "pnpm run \"/^build:.*/\" ",
+		"build:doc": "typedoc",
+		"build:pkg": "dts-buddy",
+		"clean": "pnpm run \"/^clean:.*/\"; rm -rf \"./**/node_modules\" ",
+		"clean:cache": "rm -rf \"./node_modules/.cache\" ",
+		"clean:doc": "rm -rf \"./docs\" ",
+		"clean:test": "rm -rf \"./coverage\" ",
+		"clean:types": "rm -rf \"./types\" ",
+		"dev": "pnpm run \"/^(doc:watch|dev:watch)/\" ",
+		"doc:watch": "typedoc --watch",
+		"lint": "prettier --check . && eslint .",
+		"format": "prettier . --write && eslint . --fix",
+		"check": "tsc --noEmit",
+		"typos": "typos --verbose",
+		"typos:fix": "typos --verbose --write-changes",
+		"prepare": "pnpm build:pkg",
+		"serve:doc": "serve \"./docs\" ",
+		"test": "vitest",
+		"test:ui": "vitest --ui"
+	},
+	"dependencies": {
+		"@hono/node-server": "^1.13.7",
```

Comment:
Is `@hono/node-server` unnecessary when used with Deno or Bun?

Reply:
Yes.

Reply:
However, I think this seems fine as is. There’s probably no easy way to write environment-specific dependencies.

Reply:
Oh right. Good thinking. AFAIK, that's only doable inside `package.json#exports or `package.json#imports`.

I couldn't write tests for Bun and Deno runtime as well. I never had an opportunity to write cross-runtime friendly library 🤔